webpack/assets/javascripts/react_app/components/Editor/Editor.fixtures.js

```diff
@@ -0,0 +1,56 @@
+import { noop } from '../../common/helpers';
+
+export const editor = {
+  name: 'editor',
+  diffViewType: 'split',
+  value: 'value',
+  renderedValue: '',
+  mode: 'Ruby',
+  theme: 'Monokai',
+  keyBinding: 'Default',
+  selectedView: 'input',
+  editorName: 'editor',
+  isMaximized: false,
+  isMasked: false,
+  isRendering: false,
+  readOnly: false,
+};
+
+export const dropdowns = {
+  modes: ['Text', 'Json', 'Ruby', 'Html_ruby', 'Sh', 'Xml', 'Yaml'],
```

Comment:
why is this defined twice? just wondering why we need it in the rails part to begin with?

Reply:
these are just the fixtures for testing. But the real data do come from the rails, do we prefer it in the client?

Reply:
if its static list, then yes?

Reply:
import the fixtures from the const file? (instead of defining twice?)